CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 ## [Unreleased]
 ### Changed
 * Flaw URLs consists preferably of CVE ID if possible (OSIDB-2018)
+* Changed layout on Descriptions, References and Acknowledgements
+* Renamed `Description` (Comment#0) and `Summary` (Description) based on OSIDB API logic
+* Made text area fields visible with empty content
+* Minor `FlawForm` style adjustments
+
+### Fixed
+* Fixed overlapping on edit buttons of `EditableList` items
+* Fixed inconsistent style on `LabelStatic` when model value is empty
```

Comment:
I think changelog's main focus should be user facing changes and thus they should be probably described with "non-developer" vocabulary. Eg. terms like `LabelStatic` or `EditableList` are not very suitable as user might (and probably will) have no idea what is that referring to.

Reply:
That has lot of sense, I agree. Thanks for clarifying.